package/kernel/mac80211/ath.mk

```diff
@@ -98,7 +98,7 @@ define KernelPackage/ath/config
 		bool "Atheros wireless debugging"
 		help
 		  Say Y, if you want to debug atheros wireless drivers.
-		  Only ath9k & ath10k make use of this.
+		  Only ath9k & ath10k & ath11k make use of this.
```

Comment:
The upstream Kconfig entry says this:
````
config ATH_DEBUG
	bool "Atheros wireless debugging"
	help
	  Say Y, if you want to debug atheros wireless drivers.
	  Right now only ath9k makes use of this.
````
In the source code it also looks like only ath9k is using this.

Did you notice any changes in ath11k when this is activated?

I think we should remove ath10k from this comment.

Reply:
Sorry I think you are right, the OpenWrt kconfig option `CONFIG_PACKAGE_ATH_DEBUG` is different from the kernel `ATH_DEBUG` option.